src/org/usfirst/frc/team1024/robot/commands/Auto.java

```diff
@@ -24,5 +26,6 @@ public Auto() {
         // e.g. if Command1 requires chassis, and Command2 requires arm,
         // a CommandGroup containing them would require both the chassis and the
         // arm.
+    	
```

Comment:
If this Auto CommandGroup doesn't do anything, then either delete it or remove it from this pull request.